src/window/linux/linux_window_helper.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <nox/vec.h>
+
+struct _XDisplay;
+
+namespace NOX{
+struct WindowDescriptor;
+
+namespace LinuxWindowHelper {
+
+Vector2D<int32_t> getWindowPosition(const WindowDescriptor &descriptor, _XDisplay* display);
+
+} // namespace WindowsWindowHelper
```

Comment:
LinuxWindowHelper*

These `namespace` comments will be inserted automatically after running clang-format so there is no need to write them manually.